src/components/Tasks/ColoredRadio.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import { makeStyles } from '@material-ui/core/styles';
+import * as colors from '@material-ui/core/colors';
+import Radio from '@material-ui/core/Radio';
+import { Omit } from '@material-ui/types';
+import FormControlLabel, { FormControlLabelProps } from '@material-ui/core/FormControlLabel';
+
+interface Props {
+  color: string;
+  label: string;
+}
+
+const useStyles = makeStyles({
+  root: {
+    color: (props: Props) => colors[props.color][600],
```

Comment:
Why do this rather than just have color be the actual color? As in `#RRGGBBAA`? You can then pass the correct color values from the outside. After you do this, it also becomes a more generic widget and should therefore be moved under `widgets`.